response_operations_ui/templates/collection_exercise/ce-header-section.html

```diff
@@ -0,0 +1,24 @@
+<h1 class="jupiter" name="page-ce-title">{{survey.shortName}} {{ce.exerciseRef}}</h1>
+<dl class="ce-info" id="ce-attributes">
+    <dt class="ce-status__title item-headlines-term ce-status">
+        Status:
+    </dt>
+    <dd class="ce-status__data item-headlines-definition ce-status" id="ce_status">
+        {{ ce.state }}
+    </dd>
+    {% if ce.state|upper == "LIVE" %}
+    <dt class="ce-status__title item-headlines-term ce-status">
+        Report:
+    </dt>
+    <dd class="ce-status__data item-headlines-definition ce-status" id="ce_status">
+        <a class="download" href="{{ url_for('collection_exercise_bp.response_chasing', ce_id=ce.id, survey_id=survey.id) }}">Response chasing</a>
+    </dd>
+    {% endif %}
+</dl>
+{% if show_set_live_button %}
+<div>
+    <form id="form-ready-for-live" action="" class="form" method="post" enctype="multipart/form-data">
+        <button id="btn-ready-for-live" onclick="return confirm('There\'s no going back...')" name="ready-for-live" class="btn" type="submit">Set as ready for live</button>
```

Comment:
a bit out of scope... but this message is quite funny. Also perhaps a hangover from before we productionised our system.

Reply:
Yeah, this is _by far_ the part of my least favourite part of the frontend code.  I've raised a ticket to replace it with a non-js reliant improvement.